tee-worker/litentry/core/assertion-build/src/a4.rs

```diff
@@ -35,8 +39,19 @@ use std::{
 // ERC20 LIT token address
 const LIT_TOKEN_ADDRESS: &str = "0xb59490aB09A0f526Cc7305822aC65f2Ab12f9723";
 
-pub fn build(identities: Vec<Identity>, from_date: String, min_balance: f64) -> Result<()> {
+pub fn build(
+	identities: Vec<Identity>,
+	from_date: ParameterString,
+	min_balance: u128,
+	shard: &ShardIdentifier,
+	who: &AccountId,
+	bn: ParentchainBlockNumber,
+) -> Result<Credential> {
 	let mut client = GraphQLClient::new();
+	let mut flag = false;
+
+	let q_min_balance: f64 = (min_balance / (10 ^ 12)) as f64;
```

Comment:
LIT_TOKEN has multiple decimals: on Eth is 18, and on Substrate is 12. The calculation of q_min_balance may need to be more network related?

Reply:
what about other networks?

Reply:
Eth & BSC are 18, Litentry&Litmus are 12.

Reply:
updated